src/main/java/ch/wisv/areafiftylan/products/service/MolliePaymentService.java

```diff
@@ -131,6 +134,7 @@ public Order updateStatus(String orderReference) {
                     }
                     case "paid": {
                         order.setStatus(OrderStatus.PAID);
+                        slackNotificationService.sendSlackMessage(order.getTickets().size() + " tickets bought!");
```

Comment:
We may need a :tada: or :money_with_wings: emoji in here

Reply:
Implemented in the Aspect!